libcxx/test/std/algorithms/ranges_projection_not_called_more_than_three_times.pass.cpp

```diff
@@ -0,0 +1,17 @@
+#include <algorithm>
```

Comment:
As a general note, we require a license at the top of our files (headers, source files, etc.).

However, in this case I think you should add the new test inside `libcxx/test/std/algorithms/alg.sorting/alg.clamp/ranges.clamp.pass.cpp` with a comment like `// Make sure we don't call the projection more than three times per [alg.clamp], see #64717`.